Updating our newsletter strategy

andyvan-ph opened the pull request:
## Changes

Updated the handbook with rough outline of what I'd like to do with a new newsletter. It's still quite top-level, but looking for comments for what people would like to see from a regular marketing email.

As per the update, my view is it should be primarily aimed at existing users.

## Checklist
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)
- [x] Words are spelled using American English
- [x] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

andyvan-ph commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -4,22 +4,24 @@ sidebar: Handbook
+
 - Newsletter signups from the website
 - PostHog users who opt-in
 - Event attendees
```
Are we actually doing all of these? From what I can tell, there is no opt-in procedure during account sign-up and I'd be surprised if we actually captured emails from Eventbrite from the recent event.

charlescook-ph commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -4,22 +4,24 @@ sidebar: Handbook
+1. PostHog Array, a product announcement email published every four weeks
+2. PostHog News (working title), a marketing and content email published every two weeks
+
+Both emails are sent via Mailchimp. PostHog News (working title) is a work-in-progress newsletter which we plan to increase to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. PostHog News is primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, ...
```
Which metric of ours is PostHog News intended to help? I think knowing that will help us focus when in the future we're trying to decide what content to include or not include (e.g. could be Product Discoveries).

andyvan-ph replied:
Good question. I think influencing discoveries would be hard to attribute. I think the simplest metric is just 'pageviews'. If our users are reading our content, be it tutorials, they'll either be learning something new or engaging more with us as a brand. CTR as measured in Mailchimp (do we have a Mailchimp plug-in?) would be a good quality metric, as wouuld open rates etc.

andyvan-ph replied:
Yeah, that all makes sense. Discoveries feels like a sensible option.

jamesefhawkins replied:
(Good discussion)

I'd have assumed more word of mouth growth would be our goal here.

However, we view this as an expansion of usage exercise?

If so feels like we need way more tutorial stuff in the calendar...

andyvan-ph replied:
I think it sits in both, really. My view is we can deliver a newsletter that our users enjoy and find useful, it's another means of making them into advocates for us.

charlescook-ph replied:
> However, we view this as an expansion of usage exercise?

I agree with that - quality content to existing users/newsletter subscribers (who may not be users) should increase their in-org usage, either through recipients spending more time in the product or adding more users within their org. 

@andyvan-ph correct me if I'm wrong, but I believe the intention is to have every newsletter include a tutorial (starting with something every 2 weeks until we're confident we can increase frequency).

andyvan-ph replied:
Yes, that's correct. I [have a draft](https://docs.google.com/document/d/1hknRyf5Zb6cLuSyEpnawVw6NdRBnva6bQcL3vo37cZw/edit) for comments, though maybe I should make this a PR? Let me know if you'd prefer that showing that.

Basic idea is:

- Newsletter every two weeks
- Every newsletter will feature a blog post and a tutorial
- Some curated content from elsehwhere that we think is interesting and relevant
- A fun cartoon from Lottie despiting a movie/TV show etc. to add some character

But still playing around with it and could really do with some initial impressions.

andyvan-ph commented:
I've written up a draft newsletter in Google Docs. Please critique in-depth here: https://docs.google.com/document/d/1hknRyf5Zb6cLuSyEpnawVw6NdRBnva6bQcL3vo37cZw/edit?usp=sharing

I want to try and make this feel and read like an actual email, rather than having something too designed.

charlescook-ph commented:
I like this draft a lot - I would subscribe to this newsletter! Leaving more detailed comments here as I can't in the Doc directly:

- I'm torn on the informal opening - I think I like it, but the hedgehog fact should go, it's extra verbiage that gets in the way
- I'd open with the tutorial each time as it's probably the most important piece of content relative to our objectives
- OG image section is cool, but I'd frame it as 'this is a cool thing _you_ could do' rather than making it PostHog-centric if possible
- Really like the other more informal sections - they give it a lot of personality and make it more shareable. I'd add a link to our User Slack in the competition section in case people aren't already in there. 
- I think we can cut the jobs we're hiring for and link to our careers page to save space - it's a lower value section of the newsletter anyway

andyvan-ph commented:
Good feedback, thank you.

- In hindsight the OG images thing might be a poor example here as, in most cases, we probably wouldn't have screenshots in most newsletters.

- RE: starting with the tutorial, I think that makes sense, though I'd like a little flexibility to switch things around. For me the most important thing is there will always be one in there.

- Thoughts on the name? I'm a bit unsure about it atm but can't think of anything better

charlescook-ph commented:
> In hindsight the OG images thing might be a poor example here as, in most cases, we probably wouldn't have screenshots in most newsletters.

James' product people article could be a good one? Don't think it needs to be the freshest article, just the best one of the last couple of weeks maybe. 

> Thoughts on the name? I'm a bit unsure about it atm but can't think of anything better

Nothing better comes to mind at the moment. HogPost?

andyvan-ph commented:
I've done a second pass on the draft template:

- Removed the hedgehog fact... might throw it in at the bottom instead as a fun footer?
- Changed the blog post example to something without images, which feels more reprasentative
- Added more to Stuff we're reading section (five seems like a good minimum)
- Removed the ICYMI and added more blog links to the last section, now called 'Community and updates'

Linky: https://docs.google.com/document/d/1hknRyf5Zb6cLuSyEpnawVw6NdRBnva6bQcL3vo37cZw/edit?usp=sharing

tiina303 commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -32,6 +34,16 @@ Our current newsletter structure looks roughly as follows:
-Note: Within the Mailchimp dashboard, users are automatically tagged to indicate where they came from e.g. Newsletter Subscribers, Deployed PostHog, Eventbrite, etc.
+## PostHog News format (working title)
+
+The format of the newsletter is still in development, but could/should include the following elements:
+
+- A new piece of fun hedgehog artwork
+- Featured content comprising at least one blog post and one tutorial
+- Tip/idea of the week to aid discovery  
+- PostHog news and product updates, including any updates we've made to documentation
+- Requests for comments on open PRs and issues we'd like feedback on1
```
```suggestion
- Requests for comments on open PRs and issues we'd like feedback on
```

andyvan-ph replied:
```suggestion
HogMail roughly follows this format:

- Featured content comprising at least one blog post and one tutorial
- Curated third-party content we think will interest our users 
- PostHog news, feedback requests and community updates
- The Needle comic / cartoon from Lottie
```

andyvan-ph commented:
FYI, I've sent everyone a test email of what I've hacked together in Mailchimp.

After a convo with Lottier, we've decided to make the email from 'Max - Hedgehog in Residence' complete with a little Max avatar, which Lottier is working on atm.

I've also changed up the top half a little. I've include two tutorials and changed the formatting to tighten things up, provide more context and make the whole thing more scannable.

If I can work out a way to share the design in here, I will!

andyvan-ph commented:
Sod it, I just stitched two images together.

![hogmail](https://user-images.githubusercontent.com/92976667/157928658-14d57087-3d37-425c-bff2-0020da49362d.png)

andyvan-ph commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -4,22 +4,24 @@ sidebar: Handbook
 
-Speak to Charles, Yakko, or Charles for more information. 
+1. PostHog Array, a product announcement email published every four weeks
+2. PostHog News (working title), a marketing and content email published every two weeks
```
```suggestion
2. HogMail, a marketing and content email published every two weeks
```

andyvan-ph commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -4,22 +4,24 @@ sidebar: Handbook
+1. PostHog Array, a product announcement email published every four weeks
+2. PostHog News (working title), a marketing and content email published every two weeks
+
+Both emails are sent via Mailchimp. PostHog News (working title) is a work-in-progress newsletter which we plan to increase to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. PostHog News is primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, ...
```
```suggestion
Both emails are sent via Mailchimp. We plan to move HogMail to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. It's primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, not user acquisition.
```

andyvan-ph commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -32,6 +34,16 @@ Our current newsletter structure looks roughly as follows:
+For reference, here's an [example](/blog/the-posthog-array-1-32-0) of a previous newsletter.
 
-Note: Within the Mailchimp dashboard, users are automatically tagged to indicate where they came from e.g. Newsletter Subscribers, Deployed PostHog, Eventbrite, etc.
+## PostHog News format (working title)
```
```suggestion
## HogMail format
```

andyvan-ph commented on contents/handbook/growth/marketing/newsletter.md:
```diff
@@ -32,6 +34,16 @@ Our current newsletter structure looks roughly as follows:
+- Tip/idea of the week to aid discovery  
+- PostHog news and product updates, including any updates we've made to documentation
+- Requests for comments on open PRs and issues we'd like feedback on1
+- Community section highlighting and celebrating community contributors
+- Promotion of upcoming events, virtual and in-person
```
```suggestion
```
